client/src/components/RatingsAndReviews/ProductBreakdown.jsx

```diff
@@ -1,9 +1,22 @@
 import React from 'react';
+import ReviewStars from '../common/ReviewStars';
+
+const ProductBreakdown = ({reviewMeta}) => {
+  const ratings = reviewMeta.ratings;
+  const ones = parseInt(ratings["1"]);
+  const twos = parseInt(ratings["2"]);
+  const threes = parseInt(ratings["3"]);
+  const fours = parseInt(ratings["4"]);
+  const fives = parseInt(ratings["5"]);
+  const totalScore = ones + twos*2 + threes*3 + fours*4 + fives*5;
+  const totalRatings = ones + twos + threes + fours + fives;
+  const average = Math.round(totalScore/totalRatings*10)/10;
```

Comment:
.toFixed might've also worked here

Reply:
I actually was thinking of doing that but would have been my first time using it, ill try it